Doc/library/sys.rst

```diff
@@ -11,6 +11,31 @@ interpreter and to functions that interact strongly with the interpreter. It is
 always available. Unless explicitly noted otherwise, all variables are read-only.
 
 
+.. data:: abi_info
+
+   A :term:`named tuple` holding information about the ABI of the interpreter.
+
+   .. attribute:: abi_info.pointer_bits
+
+      The width of pointers in bits, as an integer.
+
+      * ``32``: 32-bit build
+      * ``64``: 64-bit build
+      * ``None`` if this information is unknown or neither 32 nore 64 bits.
```

Comment:
IMO, for different pointer sizes we should simply use different numbers.

To me, `None` would mean something like an ABI that doesn't use pointers at all. (In which case we don't have  a C-style ABI at all, and `abi_info` itself should be missing.)